### PR TITLE
Tutorial added for LOJ 1425 - The Monkey and the Oiled Bamboo (en, bn)

### PR DESCRIPTION
**Pull request by subhashiscse**

### CODE REVIEW

**Comment by subhashiscse** [comment]
Explanation added with a new branch.

**Comment by subhashiscse** [comment]
@rebornplusplus   can you please check my other tutorial?

**Comment by subhashiscse** [comment]
This tutorial have been updated with explanation. You may check again now. @rebornplusplus

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Your solution has a bug. Consider this input:
```
1
2
9 10
```
Output should be 9. Produces 1.

**Comment by rebornplusplus** [comment]
@jan876 I think 1425 has weak dataset. Consider this submission: https://lightoj.com/submission/2079589

**Comment by subhashiscse** [comment]
After updating the data set may be my first code is not working.
Now i update the code and description. It works fine.
Please check now.

File: 1425/en.md [hunk 1]
```diff
@@ -23,50 +22,42 @@ If you are still stuck with this problem, check the codes below:
 
 ```c++
 #include<bits/stdc++.h>
+#define fr(i1,m) for(int i1=0;i1<m;i1++)
```
Comment (rebornplusplus) [inline 1425/en.md:25]: Can you please remove this define? Improves readability.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Binary search was quite alright actually. You just had to mind the first step as well. Anyway, now the algorithm has a better runtime. Kudos.

**Comment by subhashiscse** [comment]
Solved. Now i think its better to user readable.
Thanx a lot.

**Review by rebornplusplus — APPROVED** [review]
Thank you for your contribution.

**Comment by subhashiscse** [comment]
Hello brother,
Tutorial is not visible here.When it'll visible?
https://lightoj.com/problem/the-monkey-and-the-oiled-bamb

**Comment by rebornplusplus** [comment]
Dunno. The process is manual I think. Usually takes a day or two.

**Pull request merged**